wiremock-stop.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [ -f wiremock.pid ]; then
+  if kill -0 $(cat wiremock.pid) 2>/dev/null; then
+    kill $(cat wiremock.pid) && rm wiremock.pid
+  else
+    rm wiremock.pid
```

Comment:
if there wasn't a process with that id running (as something went wrong last time) then just delete the spurious pid file.